### PR TITLE
fix: remove playground results flip

### PR DESCRIPTION
**Pull request by kwasniew**
## About the changes

Before:
* when try configuration was clicked in playground and the results fetching was over there was a quick moment where playground guidance/instructions showed up before the result table was rendered

After:
* instruction if only shown before form submission
* after form submission loaded appears
* when loading finishes table starts rendering

### CODE REVIEW

File: frontend/src/component/playground/Playground/AdvancedPlayground.tsx [hunk 1]
```diff
@@ -57,6 +57,7 @@ export const AdvancedPlayground: VFC<{
     const [searchParams, setSearchParams] = useSearchParams();
     const searchParamsLength = Array.from(searchParams.entries()).length;
     const { evaluateAdvancedPlayground, loading } = usePlaygroundApi();
+    const [hasFormBeenSubmitted, setHasFormBeenSubmitted] = useState(false);
```
Comment (kwasniew) [inline frontend/src/component/playground/Playground/AdvancedPlayground.tsx:60]: new variable to store if form was submitted

File: frontend/src/component/playground/Playground/AdvancedPlayground.tsx [hunk 2]
```diff
@@ -272,17 +275,25 @@ export const AdvancedPlayground: VFC<{
...
+                                            loading={loading}
+                                            features={results?.features}
+                                            input={results?.input}
+                                        />
+                                    }
+                                />
+                                <ConditionallyRender
+                                    condition={
+                                        !Boolean(results) &&
+                                        !hasFormBeenSubmitted
```
Comment (kwasniew) [inline frontend/src/component/playground/Playground/AdvancedPlayground.tsx:292]: this removing flipping effect

**Review by thomasheartman — APPROVED** [review]
This looks good to me! My only question would be if we could use data state to check whether the form has been submitted or not instead of introducing a new state variable, but I'm assuming you've already explored that possibility? But definitely non-blocking. Nice work! 🏆

**Comment by kwasniew** [comment]
@thomasheartman You're right. I tried to do it w/o a new state variable but couldn't get it to work as expected.

**Pull request merged**